projects/components/src/form/form-select/form-select.component.spec.ts

```diff
@@ -17,6 +17,15 @@ export class VcdFormSelectWidgetObject extends WidgetObject<FormSelectComponent>
         return this.findElement('select').nativeElement;
     }
 
+    get clrIcon(): HTMLElement {
```

Comment:
We avoid returning an HTML element from a public method because that encourages querying outside the widget object wrapper. You probably want to return its [shape attribute](https://clarity.design/icons) as a string and you want to test that it has the correct icon

Reply:
Changed to return shape attribute.